calcom/cal.com
# feat: member management for platform

## What does this PR do?



- Adds a members tab to platform dashboard for member management (add, remove or filter someone from the team)



https://github.com/user-attachments/assets/f2661e0f-53c1-4e76-b309-18f4e6df72b7

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
This can be tested in the webapp through the following steps:
1. Create a platform team via `/settings/platform/new` page
2. In the platform dashboard in `/settings/platform/` page you'll see a members tab, click that tab
3. Try to add a new member or delete one in `/settings/platform/members`
